modules/ROOT/pages/security.adoc

```diff
@@ -2,23 +2,15 @@
 include::_partials/attributes.adoc[]
 // end {partialsdir}/attributes.adoc[]
 
-= Security Services
+= Making your mobile app secure
 
-There are two security services available:
+This guide does not address all mobile security topics. See <link> for an introduction to mobile security.
+{org-name} helps you achieve security using:
 
+* {app-security-service} service - remotely disable specific app variants
+* {device-security-service} - for example, disallow running app on rooted devices
```

Comment:
Could we link to the Device Security Section from here? And vice versa in a similar heading in the other section?